### PR TITLE
refactor dependencies + tests 

### PR DESCRIPTION
**Pull request by zoepage**
- added standard, formated moment.parseFormat.js
- bower fast raus
- added coveralls
- tape + tests are pending

### CODE REVIEW

File: moment.parseFormat.js [hunk 1]
```diff
@@ -4,213 +4,212 @@
   /* global define */
```
Comment (gr2m) [inline moment.parseFormat.js]: You can remove lines 5-15 entirely, and start with something like

```js
module.exports = parseFormat

function parseFormat() {
  var dayNames =  [ 'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
  // etc
}
```

Then at the end check if `global.moment` is defined, and if it is, add it like this

```ls
if (global.moment) {
  global.moment.parseFormat = parseFormat()
}
```

Does that make sense?

Comment (gr2m) [inline moment.parseFormat.js]: Or maybe better even do it like the PouchDB folks do it in their plugins:
https://github.com/nolanlawson/pouchdb-all-dbs/blob/master/index.js#L148-L151

**Comment by gr2m** [comment]
can you please merge https://github.com/gr2m/moment.parseFormat/pull/29 & rebase your PR on master

**Comment by gr2m** [comment]
done :ok_hand: I've also removed .jshintrc & Gruntfile.js

**Comment by zoepage** [comment]
perfect, thank you!
:tada:

**Pull request merged**